commons/connector/altoida/upload_altoida_acceleration.avsc

```diff
@@ -0,0 +1,13 @@
+{
+  "namespace": "org.radarcns.connector.altoida.AltoidaAcceleration",
```

Comment:
Can we remove the name part at the end from these schemas? e.g. `AltoidaAcceleration` which is part of the `name`. 
I checked the other schemas and it looks like their namespace does not contain this element. If I remember correctly, we use the namespace for package name of the Java files that get generated. In that case, all of this will be in different packages.